presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestAnalyzer.java

```diff
@@ -469,6 +469,14 @@ public void testGroupByWithQualifiedName3()
     }
 
     @Test
+    public void testGroupByWithDistinct()
+            throws Exception
+    {
+        // TODO: verify output
+        analyze("SELECT distinct(a, b), c FROM t1 GROUP BY t1.a, t1.b, t1.c");
```

Comment:
No. As I mentioned above, the issue has nothing to do with DISTINCT. It's due to a row value expression showing up in a GROUP BY query. The following query is sufficient to reproduce it:

```sql
SELECT (a, b) FROM t GROUP BY a,b
```